cristin-import/src/main/java/no/unit/nva/cristin/lambda/PublicationUpdater.java

```diff
@@ -212,4 +264,10 @@ private static AssociatedArtifactList updatedAssociatedLinks(
         list.addAll(incomingAssociatedLinks);
         return new AssociatedArtifactList(list.stream().distinct().toList());
     }
+
+    private static <T> boolean listShouldBeOverwritten(List<T> oldList, List<T> newList) {
+        return !(oldList.size() == newList.size()
+               && new HashSet<>(oldList).containsAll(newList)
+               && new HashSet<>(newList).containsAll(oldList));
```

Comment:
Isn't this enough?

```suggestion
        return !oldList.equals(newList);
```

Reply:
Tror ikke det nei, i det tilfelle må to lister være identiske, men i det her tilfelle antar jeg at de er identiske hvis de inneholder samme entries, uansett rekkefølge.

Reply:
Ah, okei 👍